platform/android/display_server_android.h

```diff
@@ -91,6 +98,10 @@ class DisplayServerAndroid : public DisplayServer {
 
 	void _set_key_modifier_state(Ref<InputEventWithModifiers> ev);
 
+	int _button_index_from_mask(int button_mask) const;
```

Comment:
This method can be `static`.

Reply:
I know, but if it can be static, it should? Genuinely ask

Reply:
In this case it would make it clear that it's a helper method that doesn't access any member.